### ISSUE TITLE
[feat] webview window will be blocked by the computer tab bar.

### ISSUE THREAD

**Issue by li-yu-tfs** [tauri-apps/tauri#11393]
### Describe the problem

this is my code 
`    if let Some(primary_monitor) = app_handle.primary_monitor().unwrap() {
        let screen_size = primary_monitor.size();
        let window_size = voice_bubble.inner_size().unwrap();
        let margin = 100.0; 

        let x = screen_size.width as f64 - window_size.width as f64 - margin;
        let y = screen_size.height as f64 - window_size.height as f64 - margin;

        let primary_position = primary_monitor.position();
        
        voice_bubble
            .set_position(tauri::Position::Physical(tauri::PhysicalPosition {
                x: (primary_position.x as f64 + x) as i32,
                y: (primary_position.y as f64 + y) as i32,
            }))
            .unwrap();
    }`

because i will set this window to my computer right bottom. but our computer ,windows or mac , have tab bar, for exmaple

<img width="1007" alt="image" src="https://github.com/user-attachments/assets/0dba2152-ac05-4d1e-8ca5-922ca4afd5b4">

so , if we set webview window to right bottom , our window will be blocked by the computer tab bar.

how to resolve this issue ? thanks for answer.

### Describe the solution you'd like

i hope have give me methods.

**Comment by FabianLars** [comment]
I'd consider this a duplicate of https://github.com/tauri-apps/tauri/issues/6592
